contrib/pylightning/tests/test_units.py

```diff
@@ -0,0 +1,31 @@
+from lightning import Millisatoshi
+import pytest
```

Comment:
flake8 is upset:

    contrib/pylightning/tests/test_units.py:2:1: F401 'pytest' imported but unused

Reply:
Whoops. Fixed